content/about.md

```diff
@@ -274,6 +274,89 @@ The title says it all.
 <!-- /marekt -->
 
 
+### Bardaci
+-----------
+
+The name **Bardaci** is just garbled Slovak word 'babraci', which we can
+translate as 'numb hand'. Although that's not always true, we have used it
+since Mr. Gazi called so the first original **Bardaci** (those two below).
+So it's really hard to describe one without the other and vice versa, they have
+come neck by neck and both are similarly crazy.
+
+But give it a try:
+
+<!-- adamh -->
+<div class="photo" markdown="1">
+
+![Herda](/img/avatars/adam_h.png)
+
+</div>
+
+<div class="about" markdown="1">
+#### Adam Herda {#herda}
+*Bardak 1*
```

Comment:
@vargac Please put there *master of something*. I believe you can think up something cool